src/tests.rs

```diff
@@ -5,7 +5,9 @@ use crate::{
 };
 use anyhow::{bail, Result};
 use serial_test::serial;
-use std::sync::{Arc, Mutex};
+// use std::sync::{Arc, Mutex};
```

Comment:
Why we want to keep this comment ?

Reply:
You're right! I've removed it and in the mean time, I've fixed a clippy warning.